templates/scenarios/ts/office-addin/teamsapp.yml

```diff
@@ -0,0 +1,3 @@
+version: 1.0.0
```

Comment:
@1yefuwang1 please review the changes for office-addin template

Reply:
@1yefuwang1 I will merge this PR first. Let me know if you have any comments and I will create new PR to resolve your comments.